### PR TITLE
FEAT empty state component

### PR DESCRIPTION
**Pull request by MarcFairbrother**
## Description

Adds a new empty state component for use in a section or in a page with the `.mod-page` modifier.

Adds a `luSafeExternalSvg` pipe to inject the code from a remote SVG file into the DOM. Use this for the icons which have css custom properties for theming reasons.

-----

```
<div class="emptyPage-content-icon"
	aria-hidden="true"
	[innerHtml]="'https://my-file.svg' | luSafeExternalSvg"></div>
```

### Full Page

The following custom properties should be used to handle the background illustrations:

```
--components-emptyState-illustration-background-bottom-left: url('https://my-file.svg');
--components-emptyState-illustration-foreground-bottom-left: url('https://my-file.svg');
--components-emptyState-illustration-background-top-right: url('https://my-file.svg');
--components-emptyState-illustration-foreground-top-right: url('https://my-file.svg');
```

### To Do

- ~~svg icons need to be uploaded to the CDN after realignment~~
- ~~resolve naming conflict with legacy empty state component~~
- ~~rename css classes as `emptyState` instead of `emptyPage`~~

### Section

chrome_DOgNpxkVrq

### Full Page

chrome_25avuWkkvf

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2369/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/8106/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2369/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2369/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9389/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2369/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9391/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2369/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/9397/ui-diff)

File: packages/scss/src/components/emptyPage/vars.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@mixin vars {
+	--components-emptyState-background-color: var(--commons-background-base);
+	--components-emptyState-container-width: var(--components-container-max-width);
+	--components-emptyState-illustration-top-right-offset-left: min(calc(100% + 5rem), 91.5rem);
+	--components-emptyState-illustration-top-right-offset-left-S: min(calc(100% + 3.5rem), 91.5rem);
+	--components-emptyState-illustration-background-bottom-left: url('https://cdn.lucca.fr/lucca-front/assets/empty-states/lucca/bubbles-bottom-left-01.svg');
+	--components-emptyState-illustration-foreground-bottom-left: url('https://cdn.lucca.fr/lucca-front/assets/empty-states/lucca/lucca-01.svg');
+	--components-emptyState-illustration-background-top-right: url('https://cdn.lucca.fr/lucca-front/assets/empty-states/lucca/bubbles-top-right-01.svg');
+	--components-emptyState-illustration-foreground-top-right: url('https://cdn.lucca.fr/lucca-front/assets/empty-states/generic/coffee-01.svg');
```
Comment (MarcFairbrother) [inline packages/scss/src/components/emptyPage/vars.scss:9]: Je me demande si on doit inclure des illustrations par défaut dans le composant ou si ces variables ne devraient pas plutôt être renseignées au niveau de la story.

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2369/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/10900/ui-diff)

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11015/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2369/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11019/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2369/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11021/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2369/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11074/ui-diff)

**Review by jeremie-lucca — APPROVED** [review]

**Review by vvalentin-lucca — APPROVED** [review]

**Review by GuillaumeNury — APPROVED** [review]

**Pull request merged**